python/Makefile.am

```diff
@@ -14,7 +14,7 @@ all-local:
 	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV)
 
 install-exec-local:
-	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV) install --root ${DESTDIR} --prefix=${prefix}
+	CFLAGS="${CFLAGS} -fno-strict-aliasing" @PYTHONBINARY@ setup.py build --build-base py$(PYTHONREV) install --root ${DESTDIR} --prefix=${prefix} --install-lib=${PYTHONSITEDIR}
```

Comment:
`pythonsitedir` is lowercase above as make variable - should it be overridden?
This does not seem right.

Reply:
Interesting. This worked in my testing with cross-build (in https://lists.openembedded.org/g/openembedded-devel/message/108217), but you're right lowercase pythonsitedir makes more sense here, will retest with OE and update.